src/System.Private.ServiceModel/tools/test/Bridge/Bridge/ensureBridge.ps1

```diff
@@ -24,7 +24,9 @@ $result= checkBridge;
 if(!$result)
 {
 	Write-Host Launching bridge.exe.
-	$bridgePath = Join-Path $PSScriptRoot bridge.exe
+
+	$bridgePath = Join-Path $PSScriptRoot ..\bin\bridge.exe
+
 	Start-Process $bridgePath $portNumber -WorkingDirectory $PSScriptRoot
```

Comment:
Sorry about the delay. I believe Matt is correct. The working directory was set here so that if you passed a relative path into the config call it would look in the right place.